docs/user/dashboard/lens.asciidoc

```diff
@@ -54,6 +54,27 @@ For array value fields, the percentage distribution considers each value in the
 
 TIP: *Other* can equal more than 100% by a small amount.
 
+[float]
+[[add-fields-in-lens]]
+===== Add fields
+
+Add and define fields that you want to visualize using the {ref}/modules-scripting-painless.html[Painless scripting language].
```

Comment:
IMHO we should highlight the field is not local to the visualization but saved on the index pattern, so it will show up in other places which are working with the same index pattern.